apisix/cli/ngx_tpl.lua

```diff
@@ -765,6 +765,18 @@ http {
             }
             {% end %}
 
+
+            {% if proxy_mirror_timeouts then %}
+                {% if proxy_mirror_timeouts.connect then %}
+            proxy_connect_timeout {* proxy_mirror_timeouts.connect *};
+                {% end %}
+                {% if proxy_mirror_timeouts.read then %}
+            proxy_read_timeout {* proxy_mirror_timeouts.read *};
+                {% end %}
+                {% if proxy_mirror_timeouts.send then %}
+            proxy_send_timeout {* proxy_mirror_timeouts.send *};
+                {% end %}
+            {% end %}
```

Comment:
These directives apply to all plugins, I don't think it's appropriate for you to do this. Maybe you can implement this functionality in your own plugin by `before_proxy`.

Reply:
No, there is a guard above:
```
{% if enabled_plugins["proxy-mirror"] then %}
```

Reply:
@soulbird No, I think this is a location level configuration and only works for /proxy_mirror internal request